docs/koji.md

```diff
@@ -55,6 +55,7 @@ The index map has these entries:
 
 - `pull` (str list): docker pull specifications for the manifest list, by tag and by digest
 - `tags` (str list): tags applied to the manifest list when it was created
+- `digests` (map): a dict of media types and digests of the image
```

Comment:
https://docs.pagure.org/koji/content_generator_metadata/ has this:

> digests: A map of media type (such as “application/vnd.docker.distribution.manifest.v2+json”) to manifest digest (a string usually starting “sha256:”), for each available media type.

I'd like to avoid "digests of the image" since it's the manifest in particular that it's a digest of.

Reply:
Good idea, updated